src/Command/Analyze.php

```diff
@@ -652,33 +766,81 @@ private function analyzeFailures(string $test, string $parser, bool $justAgents
+        \$table->setColumnWidth(0, 50);
+        \$table->setColumnMaxWidth(0, 50);
+        \$table->setColumnWidth(1, 50);
+        \$table->setColumnMaxWidth(1, 50);
+        \$table->setColumnWidth(2, 50);
+        \$table->setColumnMaxWidth(2, 50);
+        \$table->setStyle('box');";
+
         $table->setHeaders([
             [new TableCell('UserAgent', ['colspan' => 3])],
-            ['Browser', 'Platform', 'Device'],
+            [new TableCell('Browser'), new TableCell('Platform'), new TableCell('Device')],
         ]);
 
+        $output .= "
+        \$table->setHeaders([
+            [new TableCell('UserAgent', ['colspan' => 3])],
+            [new TableCell('Browser'), new TableCell('Platform'), new TableCell('Device')],
+        ]);
+        
+        \$rows = [];
+        ";
+
         $rows = [];
         foreach ($this->failures[$test][$parser] as $agent => $failData) {
-            $rows[] = [new TableCell((string) $agent, ['colspan' => 3])];
-            $rows[] = [
-                isset($failData['browser']) ? $this->outputDiff($failData['browser']) : '',
-                isset($failData['platform']) ? $this->outputDiff($failData['platform']) : '',
-                isset($failData['device']) ? $this->outputDiff($failData['device']) : '',
-            ];
-            $rows[] = new TableSeparator();
-
             if ($justAgents === true) {
                 $this->output->writeln($agent);
+            } else {
+                $rows[] = [new TableCell((string) $agent, ['colspan' => 3])];
+                $rows[] = [
+                    new TableCell(isset($failData['browser']) ? $this->outputDiff($failData['browser']) : ''),
+                    new TableCell(isset($failData['platform']) ? $this->outputDiff($failData['platform']) : ''),
+                    new TableCell(isset($failData['device']) ? $this->outputDiff($failData['device']) : ''),
+                ];
+                $rows[] = new TableSeparator();
+
+                $output .= "
+        \$rows[] = [new TableCell('" . (string) $agent . "', ['colspan' => 3])];
+        \$rows[] = [
+                    new TableCell('" . (isset($failData['browser']) ? $this->outputDiff($failData['browser']) : '') . "'),
+                    new TableCell('" . (isset($failData['platform']) ? $this->outputDiff($failData['platform']) : '') . "'),
+                    new TableCell('" . (isset($failData['device']) ? $this->outputDiff($failData['device']) : '') . "'),
+                ];
+        \$rows[] = new TableSeparator();
+                ";
             }
         }
 
-        array_pop($rows);
-
-        $table->setRows($rows);
         if ($justAgents === false) {
+            array_pop($rows);
+
+            $table->setRows($rows);
+
+            $output .= "
+        \$table->setRows(\$rows);
+        \$table->render();
+            ";
+
+
             $table->render();
         }
+
+        file_put_contents('output.txt', $output);
```

Comment:
What was the plan with this file (and all of the above lines that are building out the `$output` variable?

It doesn't seem complete at the moment, as the file is never referenced outside of this write, and isn't executable by itself.

Reply:
Sorry, I forgot to remove debugging code. I will remove this.